pkg/krmtotf/references.go

```diff
@@ -232,6 +232,16 @@ func resolveTargetFieldValue(r *Resource, tc corekccv1alpha1.TypeConfig) (interf
 }
 
 func resolveDefaultTargetFieldValue(r *Resource, tc corekccv1alpha1.TypeConfig) (interface{}, error) {
+	// Resolve target value from direct resources, when spec.resourceID is unspecified.
```

Comment:
Could you update the comment to match the code? Looks like we don't care whether spec.resourceID is set or not for a direct resource, but just do `Resolve default target value from status.externalRef for direct resources.`